.github/workflows/build.yml

```diff
@@ -24,8 +24,8 @@ jobs:
       JDK_VER: 13.0.x
       DAPR_RUNTIME_VER: 0.4.0-rc.1
       DAPR_INSTALL_URL: https://raw.githubusercontent.com/dapr/cli/f84566fb2bf5a599252ab9d6bd82fc78faf94dba/install/install.sh
-      DAPR_CLI_REF:
-      DAPR_REF:
+      DAPR_CLI_REF: eb76cbf2200058f849814649f23212b3247a952c
+      DAPR_REF: 034b82ebfbb7c76e5801ae6d90c2edd29f629df8
```

Comment:
These overrides are useful for integration tests to use an unreleased version via building from source. This feature was used in 0.4 release.